hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -42,6 +42,10 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 
```

Comment:
Separate PR for the Azure changes? Think making changes to something as core as a FileSystem implementation in such a big PR is a bad idea.

Reply:
Still haven't looked at the changes here though.

Reply:
it goes with the tests. we could merge separately, but we need to get them right together